src/style.css

```diff
@@ -1,3 +1,92 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  /* width: 100%; */
+  margin-bottom: 20px;
+  width: 1024px;
+}
+
+.logo_png {
+  display: block;
+  width: 40px;
+  height: 40px;
+
+  /* position: relative; */
+
+  /* top: 10px; */
+}
+
+a.blue {
```

Comment:
The class `a.blue` is used in the HTML, but ensure that it is consistently applied and matches the intended design. Verify that all elements using this class are styled as expected.